lib_async/ocaml_async_kafka.c

```diff
@@ -224,41 +219,6 @@ CAMLprim value ocaml_kafka_async_consumer_poll(value caml_kafka_handler) {
   CAMLreturn(result);
 }
 
-CAMLprim value ocaml_kafka_async_produce(value caml_kafka_topic, value caml_kafka_partition, value caml_opt_key, value caml_msgid, value caml_msg) {
```

Comment:
Not obvious at first sight, but yes this code is a duplicate of `ocaml_kafka_produce`.